Changelog.md

```diff
@@ -19,6 +19,9 @@ Nothing here yet.
 - Remove unused `JalaliMixin`
 - Replace `pkg_resources` with `importlib_metadata` to fetch the version number in `__init__.py` (#657)
 - Added new badges (pypi, conda, license) and installation instructions (pip, conda) to readme file @sugatoray (#673).
+- Replace `pyCalverter` with `convertdate` (#536)
```

Comment:
This line and the line below are duplicated from the lines 18-19 and should be removed from your patch.